docs/usage/self-hosted-configuration.md

```diff
@@ -53,7 +53,7 @@ npx ng update @angular/core --from=9.0.0 --to=10.0.0 --migrateOnly --allowDirty
 ## allowedPostUpgradeCommands
 
 A list of regular expressions that determine which commands in `postUpgradeTasks` are allowed to be executed.
-If this list is empty then no tasks will be executed.
+If this list is empty then no tasks will be executed. Also you need to have `"trustLevel": "high"`, otherwise option will be ignored.
```

Comment:
```suggestion
If this list is empty then no tasks will be executed.
Also you need to have `"trustLevel": "high"`, otherwise option will be ignored.
```

We use the "one line per sentence standard", so please do that here as well. 😄 
I'm not 100% what `option will be ignored` refers to, can we specify what option will be ignored (I guess the option that is ignored is `postUpgradeTasks`?

Reply:
I aslo noticed we already have this:

https://github.com/renovatebot/renovate/blob/cab5d4e5906c2ce125cc9940d1817389383856b4/lib/config/definitions.ts#L102-L110

This is added to the docs/usages/self-hosted-configuration page when the website is built.
So maybe we should change the `description` field there as well/instead of what we're doing now?

Reply:
Also we have a example config given on L60 of the `docs/usage/self-hosted-configuration.md`.
Should we maybe add the `trustLevel`: "high" config here???

```json
{
    "allowedPostUpgradeCommands": ["^tslint --fix$", "^tslint --[a-z]+$"],
    "trustLevel": "high"
}
```

Reply:
Yes, let's shorten the description and put the bulk into the md file